pandas/tests/frame/test_constructors.py

```diff
@@ -241,6 +241,17 @@ def test_constructor_dict(self):
         # Corner cases
         assert len(DataFrame({})) == 0
 
```

Comment:
make new tests

Reply:
Can you be more specific?

Reply:
make these as new tests, don't add onto the existing ones (IOW give it a separate test name)